snark-verifier-sdk/src/file_io.rs

```diff
@@ -57,7 +60,8 @@ pub fn read_pk<C: Circuit<Fr>>(path: &Path) -> std::io::Result<ProvingKey<G1Affi
     // let initial_buffer_size = f.metadata().map(|m| m.len() as usize + 1).unwrap_or(0);
     // let mut bufreader = Vec::with_capacity(initial_buffer_size);
     // f.read_to_end(&mut bufreader)?;
-    let pk = ProvingKey::read::<_, C>(&mut bufreader, SerdeFormat::RawBytesUnchecked).unwrap();
+    let pk =
+        ProvingKey::read::<_, C>(&mut bufreader, SerdeFormat::RawBytesUnchecked, params).unwrap();
```

Comment:
`ProvingKey::Read` is fallible and already returns `std::io::Result` we should handle the error here:
```suggestion
        ProvingKey::read::<_, C>(&mut bufreader, SerdeFormat::RawBytesUnchecked, params)?;
```